docs/api/structures/navigation-entry.md

```diff
@@ -2,3 +2,6 @@
 
 * `url` string
 * `title` string
+* `pageState` string (optional) - A base64 encoded data string containing Chromium page state
```

Comment:
Can this be a `Buffer` instead? I assume that would be better although maybe rougher in serialization?

Reply:
You could but then you'll ask people to write their own serializers/deserializers or to use stuff like https://www.npmjs.com/package/buffer-json. I'm not sure that's worth the slightly smaller size.

Reply:
Is the `pageState` data backward compatible between Chromium versions? I wonder what happens if someone saves state to disk, then the app upgrades to a new Electron version and tries to restore a page.

Reply:
Ah, @samuelmaddock pointed me to https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/navigation_entry.h;l=126-137;drc=f83c4b8b03dcb0fb79fb3ca59c8d129c72876cda which explains a bit about backward compatibility handling